groupnews-web/src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
At logback's webpage in charter configuration it was written that firstly it tries to search for logback-test.xml so I guess I just created that file without think about the logic of it. Will correct it.